WooCommerce/Resources/Images.xcassets/icon-comment-content.imageset/Contents.json

```diff
@@ -0,0 +1,21 @@
+{
+  "images" : [
+    {
```

Comment:
Perhaps you can apply here a single scale with the pdf file

Reply:
Right! Switched from SVG to PDF and universal size here: https://github.com/woocommerce/woocommerce-ios/pull/8728/commits/26d3d0668b56c555c191dbad0d5e472baf09a64e